server/events/vcs/github_client.go

```diff
@@ -425,8 +425,18 @@ func (g *GithubClient) UpdateStatus(ctx context.Context, request types.UpdateSta
 		Context:     github.String(request.StatusName),
 		TargetURL:   &request.DetailsURL,
 	}
-	_, _, err := g.client.Repositories.CreateStatus(ctx, request.Repo.Owner, request.Repo.Name, request.Ref, status)
-	return err
+
+	// TODO: For debug purposes, remove when implementing github checks
+	if request.StatusId == "" {
+		_, _, err := g.client.Repositories.CreateStatus(ctx, request.Repo.Owner, request.Repo.Name, request.Ref, status)
+		fmt.Printf("DEBUG: Creating status check: %v", status)
+		return "1", err
+	} else {
+		fmt.Printf("DEBUG: Updating status check for: %v", status)
+		_, _, err := g.client.Repositories.CreateStatus(ctx, request.Repo.Owner, request.Repo.Name, request.Ref, status)
+		return request.StatusId, err
```

Comment:
I'm a little confused here. What's the benefit of returning the same `request.StatusId` you passed in?